app/views/claims/subjects_taught.html.erb

```diff
@@ -1,39 +1,24 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { mostly_teaching_eligible_subjects: "tslr_claim_mostly_teaching_eligible_subjects_true"}) if current_claim.errors.any? %>
+    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
     <%= form_for current_claim, url: claim_path do |form| %>
-      <%= form.hidden_field :mostly_teaching_eligible_subjects %>
       <%= form_group_tag current_claim do %>
-        <fieldset class="govuk-fieldset">
+        <fieldset class="govuk-fieldset" id="tslr_claim_eligible_subjects">
           <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
             <h1 class="govuk-fieldset__heading">
-              <%= t("tslr.questions.mostly_teaching_eligible_subjects") %>
+              <%= t("tslr.questions.subjects_taught") %>
             </h1>
           </legend>
-          <p class="govuk-body">
-            Eligible subjects are:
-          </p>
-          <ul class="govuk-list govuk-list--bullet">
-            <li>biology</li>
-            <li>chemistry</li>
-            <li>physics</li>
-            <li>computer science</li>
-            <li>languages (not including English)</li>
-          </ul>
-          <p class="govuk-body">
-            <span class="govuk-hint">
-              If you've been off on long-term leave or sick, include the time you were scheduled to teach these subjects.
-            </span>
-          </p>
-          <%= errors_tag current_claim, :mostly_teaching_eligible_subjects %>
-          <div class="govuk-radios">
-            <div class="govuk-radios__item">
-              <%= form.radio_button(:mostly_teaching_eligible_subjects, true, class: "govuk-radios__input") %>
-              <%= form.label :mostly_teaching_eligible_subjects_true, "Yes", class: "govuk-label govuk-radios__label" %>
-            </div>
-            <div class="govuk-radios__item">
-              <%= form.radio_button(:mostly_teaching_eligible_subjects, false, class: "govuk-radios__input") %>
-              <%= form.label :mostly_teaching_eligible_subjects_false, "No", class: "govuk-label govuk-radios__label" %>
+          <div class="govuk-checkboxes">
+            <% TslrClaim.eligible_subjects.each do |subject, value| -%>
+              <div class="govuk-checkboxes__item">
+                <%= check_box_tag "tslr_claim[eligible_subjects][]", subject, current_claim.eligible_subjects.include?(subject), class: "govuk-checkboxes__input", id: "eligible_subjects_#{subject}"  %>
+                <%= label_tag "tslr_claim[eligible_subjects][]", t("tslr.questions.eligible_subjects.#{subject}"), class: "govuk-label govuk-checkboxes__label", for: "eligible_subjects_#{subject}"  %>
+              </div>
+            <% end %>
+            <div class="govuk-checkboxes__item">
```

Comment:
Although it's not part of the guidance on checkboxes we may want to include a text divider like on the pattern for radios: https://design-system.service.gov.uk/components/radios/#radio-items-with-a-text-divider

It feels a bit strange having 'Not applicable' next to the eligible subjects, but feel free to ignore. @titlescreen what do you think?

Reply:
I've changed it to this now, which I think is nicer, but has caused a whole load of other fires:

![image](https://user-images.githubusercontent.com/109774/60163178-fc29fa00-97f2-11e9-843b-fae6bb35f426.png)

Reply:
Does this at all work without JS? This is mixing metaphors quite a lot. I think it makes more sense to be a checkbox rather than a radio button.

Reply:
Good catch, it didn't. I've fixed that now, which has also simplified the code a lot and helped with something I'd been butting up against all morning! On the checkbox vs radio button front, after doing both, it feels more natural clicking a radio button to reset the other values - @titlescreen what do you think?

Reply:
If you don't have JS does the radio button still make sense? Something to consider.